rhizome/bin/prep_host.rb

```diff
@@ -36,3 +37,6 @@
 # For qemu-image convert and mcopy for cloud-init with the nocloud
 # driver.
 r "apt-get -y install qemu-utils mtools"
+
+# For some providers like Mars
+r "apt-get install -y dmidecode dosfstools"
```

Comment:
I suppose his is also a case of "packages that are not installed by default...sometimes". And it's probably not a mars-ism so much as how ubuntu is prepared on the server, since IIRC we're the ones who prepared it from the upstream iso in this case. Perhaps should have a "add missing dependencies for minimal Ubuntu installs" commit.

Reply:
should I move it to a different PR?